src/vecs/client.py

```diff
@@ -96,12 +96,31 @@ def get_collection(self, name: str) -> Collection:
             CollectionNotFound: If no collection with the given name exists.
         """
         from vecs.collection import Collection
+        query = text(
+        f"""
+        select
+            relname as table_name,
+            atttypmod as embedding_dim
+        from
+            pg_class pc
+            join pg_attribute pa
+                on pc.oid = pa.attrelid
+        where
+            pc.relnamespace = 'vecs'::regnamespace
+            and pc.relkind = 'r'
+            and pa.attname = 'vec'
+            and not pc.relname ^@ '_'
+            and pc.relname = :name
+        """
+        ).bindparams(name=name)
+        with self.Session() as sess:
+            query_result: Row[Tuple[str, int]] | None = sess.execute(query).fetchone()
```

Comment:
lets remove the type hints:
```
Row[Tuple[str, int]] | None
```
on `query_result`
- to avoid having to import `Row` and `Tuple`.
- the `|` syntax was introduced in python 3.10 and we support 3.7+